content/hardware/05.pro-solutions/solutions-and-kits/opta/tech-specs.yml

```
@@ -34,8 +34,8 @@ Dimensions:
   Length: 80 mm
   Height: 90 mm
 IP Protection: IP20
-RTC: ~10days, NTP sync through ethernet
+RTC: ~10days, NTP sync through Ethernet or Wifi
 Secure element: ATECC608B
 Programming:
   Arduino programming language: Via Arduino IDEs, Arduino CLI, Arduino Web Editor
-  IEC-61131-3 as option: Ladder Diagram (LD), Function Block Diagram (FBD), Sequential Function Chart (SFC), Structured Text (ST), Instruction List (IL)
+  IEC-61131-3: Ladder Diagram (LD), Function Block Diagram (FBD), Sequential Function Chart (SFC), Structured Text (ST), Instruction List (IL)
```

This is only for the Arduino PLC IDE right?
```suggestion
  IEC-61131-3 via Arduino PLC IDE: Ladder Diagram (LD), Function Block Diagram (FBD), Sequential Function Chart (SFC), Structured Text (ST), Instruction List (IL)
```

---

Let's keep as it is just in case, have my reasons :)